### ISSUE TITLE
wrong result

### ISSUE THREAD

**Issue by lilin19890401** [xmfbit/warpctc-caffe#12]
Thanks for your contribution。When I use your code to train captach follow your steps, I can see the loss is lower and accuracy is good from log. But When I test single captach Image with captcha_prediction.cpp, the result is wrong and Unbelievable，Why? Thank you

**Comment by xmfbit** [comment]
Do you test on just one image or multiple images? If you have tested on multiple images and the results are bad, you can try to test your model on training set first.

> I can see the loss is lower and accuracy is good from log

If the results are also bad on the training set, there might be some mistakes in your test script.
